js/form/src/Dropdown.svelte

```diff
@@ -104,14 +107,10 @@
 		dispatch("blur");
 	}
 
-	function handle_focus(e: FocusEvent){
+	function handle_focus(e: FocusEvent): void{
 		dispatch("focus");
-		showOptions = !showOptions;
-		if (showOptions) {
-			filtered = choices;
-		} else {
-			filterInput.blur();
-		}
```

Comment:
Unless I'm missing something, these lines can never be reached  @dawoodkhan82?

Reply:
I think your cleanup of this makes sense. But I think these lines should've been hit. How else was the input being blurred? But I also didn't go back and test on main, so not sure.